packages/api/server/http.mts

```diff
@@ -41,23 +41,6 @@ const router = express.Router();
 
 router.use(express.json());
 
-router.options('/disk', cors());
-
-router.post('/disk', cors(), async (req, res) => {
-  let { dirname } = req.body;
-
-  try {
-    const config = await getConfig();
-    dirname = dirname || config.baseDir;
-    const entries = await disk(dirname, '.src.md');
-    return res.json({ error: false, result: { dirname, entries } });
-  } catch (e) {
-    const error = e as unknown as Error;
-    console.error(error);
-    return res.json({ error: true, result: error.stack });
-  }
-});
-
```

Comment:
I think I've removed all the disk / custom file picker implementation stuff. If you see anything else I've missed, please call it out.